docs/src/whatsnew/latest.rst.template

```diff
@@ -7,31 +7,33 @@ This document explains the changes made to Iris for this release
 (:doc:`View all changes <index>`.)
 
 
-.. dropdown:: :opticon:`alert` v3.X.X Patches
+.. dropdown:: :opticon:`report` v3.X.X Release Highlights
    :container: + shadow
    :title: text-primary text-center font-weight-bold
    :body: bg-light
    :animate: fade-in
    :open:
 
-   The patches in this release of Iris include:
+   The highlights for this major/minor release of Iris include:
+
+   * N/A
+
+   And finally, get in touch with us on `GitHub`_ if you have any issues or
+   feature requests for improving Iris. Enjoy!
 
-   #. N/A
 
+v3.X.X (DD MMM YYYY)
```

Comment:
```suggestion
NOTE: section below is a template for bugfix patches
====================================================
    (Please remove this section when creating an initial 'latest.rst')

v3.X.X (DD MMM YYYY)
```
Comparing this with the current `latest.rst`, I see that we have no initial use for this section when we are creating a new `latest.rst`.
So, this section is actually a further reference to be used _later_ -- it's' a sort of template within the template.

So, _**can we use a header like this to point this out ?**_